### PR TITLE
Add feature to press Ctrl-enter to copy package name

### PR DESCRIPTION
**Pull request by mattfwood**
Added "ctrl" as an argument to paste the name of the package using the clipboard action

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ alfy.fetch('https://api.npms.io/v2/search', {
 					},
 					cmd: {
 						subtitle: `${pkg.version} published at ${dateFormat('yyyy-dd-MM', new Date(pkg.date))} by ${(pkg.author && pkg.author.name) || pkg.publisher.username}`
+					},
+					ctrl: {
+						arg: pkg.name,
+						subtitle: `Paste Package ${pkg.name}`
```
Comment (sindresorhus) [inline index.js]: It should say `Copy package name`

**Comment by sindresorhus** [comment]
Why do you specify the the `ctrl` action both in the JS file and in the Alfred workflow?

**Comment by sindresorhus** [comment]
When pressing enter on a search result while holding Ctrl down, I get this:

screen shot 2018-03-06 at 19 29 47

**Comment by sindresorhus** [comment]
Ping @mattfwood

**Comment by mattfwood** [comment]
@sindresorhus Sorry, I wasn't getting GitHub notifications for this for some reason. Let me take another look at and remove the redundancy,

**Comment by sindresorhus** [comment]
I'm still getting the issue in https://github.com/sindresorhus/alfred-npms/pull/20#issuecomment-370766918

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: index.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ alfy.fetch('https://api.npms.io/v2/search', {
 					},
 					cmd: {
 						subtitle: `${pkg.version} published at ${dateFormat('yyyy-dd-MM', new Date(pkg.date))} by ${(pkg.author && pkg.author.name) || pkg.publisher.username}`
+					},
+					ctrl: {
+						arg: pkg.name,
+						subtitle: `Copy Package ${pkg.name}`
```
Comment (sindresorhus) [inline index.js]: This makes it sound like it's going to copy the package itself.

Let's use:

> Copy package name

File: info.plist [hunk 3]
```diff
@@ -22,10 +20,26 @@
...
+				<string></string>
+				<key>vitoclose</key>
+				<false/>
+			</dict>
 		</array>
 	</dict>
+	<key>createdby</key>
+	<string>Sindre Sorhus</string>
+	<key>description</key>
+	<string>Alfred 3 workflow to search for npm packages with npms.io</string>
```
Comment (sindresorhus) [inline info.plist]: This is filled in automatically at install time based on package.json fields.

File: info.plist [hunk 4]
```diff
@@ -110,5 +152,9 @@
 			<integer>10</integer>
 		</dict>
 	</dict>
+	<key>version</key>
+	<string>1.1.0</string>
+	<key>webaddress</key>
+	<string>https://github.com/sindresorhus/alfred-npms#readme</string>
```
Comment (sindresorhus) [inline info.plist]: This is filled in automatically at install time based on package.json fields.

File: test.js [hunk 5]
```diff
@@ -4,19 +4,21 @@ import alfyTest from 'alfy-test';
...
-			alt:
-			{
+		mods: {
+			alt: {
 				arg: 'https://www.npmjs.com/package/chalk',
 				subtitle: 'Open the npm page instead of the GitHub repo'
+			},
+			ctrl: {
+				arg: 'chalk',
+				subtitle: `Paste Package chalk`
```
Comment (sindresorhus) [inline test.js]: Needs to be updated too.

**Comment by sindresorhus** [comment]
It also needs to be documented in the readme.

**Comment by mattfwood** [comment]
@sindresorhus Sorry this has been such a saga, but this time it should be all resolved.

Sorry for trying to implement this back when I had much less experience creating Alfred workflows.

File: index.js (hunk 2 above)
Comment (sindresorhus) [reply index.js]: This is incorrectly resolved. The casing is not what I wrote and the `${pkg.name}` should not be included.

**Pull request merged**